completions/cryptsetup

```diff
@@ -11,6 +11,50 @@ _cryptsetup_device()
     _filedir
 }
 
+_cryptsetup_action()
+{
+    local ret IFS=$' \t\n'
+    _comp_dequote "${1-}" || return 1
+    local cmd=${ret:-cryptsetup}
+    local actions=$(
+        {
+            LC_ALL=C "$cmd" --help 2>&1 |
+                sed -n '/^<action> is one of:/,/^[^[:space:]]/s/^[[:space:]]\{1,\}\([^[:space:]]\{1,\}\).*/\1/p'
+            LC_ALL=C man cryptsetup 2>&1 |
+                awk '/^[[:space:]]+[[:alnum:]_]+([[:space:]]+(-[^[:space:].]+|<[^<>]+>|\[[^][]+\]|or))*$/ {print $1}'
```

Comment:
I think we should add test cases for the parsing part here like we have for dnssec-keygen and xrandr.

Reply:
Actually, with this extraction, there is no guarantee that we do not hit false positives. I just tested it with the version with v2.4.3 in Fedora 36. If we care about the false positives, maybe we should filter out them using `strings` for the result as well as used in the fallback.